### ISSUE TITLE
mate-screenshot: copy to clipboard does not work

### ISSUE THREAD

**Issue by schmolch** [mate-desktop/mate-utils#275]
#### Expected behaviour

picture in clipboard

#### Actual behaviour

clipboard is empty

#### Steps to reproduce the behaviour

it works with gnome-screenshot

#### MATE general version

1.24

#### Package version

mate-screenshot 1.24.0

#### Linux Distribution

Manjaro

**Comment by N0rbert** [comment]
Can't reproduce on Ubuntu 20.04 LTS.

**Comment by schmolch** [comment]
Right, i have tried again and i can do smaller screengrabs.
As soon as the area grabbed is getting bigger the clipboard remains empty.
I'll try to figure out the exact size i can do successfully.

**Comment by LetArch** [comment]
Yes, it's not work for mate-1.24.0. More. If opened context menu, screenshot area not working. Fix it. Pls.

**Comment by igutermes** [comment]
I have the same problem in Debian bullseye.
It only seems to happen when I pass on commands. Ex:

```
mate-screenshot -a -c
mate-screenshot -w -c
mate-screenshot -c
```

If I use `mate-screenshot -i` and click my way into copying, then the program works just fine. Perhaps the buttons use a slightly different code?

**Comment by adefaria** [comment]
The man page for mate-screenshot lacks any kind of description for -c.

**Comment by igutermes** [comment]
@adefaria is correct. -c will only be mentioned in the summary of the --help option (see [here](https://github.com/mate-desktop/mate-utils/issues/177#issuecomment-1099260347) for a peek), but is not described in the man page. 

Though the problem os copying is persistent, I have had some success in applying this [workaround](https://github.com/mate-desktop/mate-utils/issues/177#issuecomment-1101623213). I have been testing for a while now and so far so good. What does the trick is the sleep part.
Just create an executable and link it in mate-keybinding-properties, works well. 

Mate version 1.26.0.

**Comment by adefaria** [comment]
Yes, I implemented a script to suit my needs using that concept. I added in a zenity popup to save the captured area to a file. Some apps (slack, thunderbird) allow you to simply paste an image from the clipboard and other apps (voice.google.com) need a file. So I prompt to save the file and if I intend to use the image in something like voice.google.com then I'll save the file, otherwise, I don't bother. The popup times out not saving the image to a file. Also grabbed a sound of a camera click to play it to emulate taking a "screenshot".

`#!/bin/bash
timeout=10
screenshot_directory=/tmp

function yesno {
  question=$1
  
  zenity --question --text "$question" --width=300 --timeout=$timeout 2> /dev/null
}

mate-screenshot -ac
aplay ~/Conf/CameraClick.wav

file="$screenshot_directory/$(date +%F@%T).png"

if yesno "Save screenshot as $file?"; then
  xclip -selection clipboard -t image/png -o > $file
fi
`
